SparkPost/2web2ui
# FE-791 - Adds Cypress integration tests for the navigation

[FE-791](https://jira.int.messagesystems.com/browse/FE-791)

### What Changed
- Adds Cypress integration tests for the navigation - some already existed, however, adding support for dynamic rendering of navigation items based on fixtures
- Fixed a templates test that was demonstrating some flakiness

### How To Test
- Verify tests are passing
- Verify tests are testing relevant scenarios controlled by user/account data - **I am probably missing some scenarios!**

### To Do
